tcs-persistence/src/main/resources/queries/traineeInfoForConnection.sql

```diff
@@ -14,6 +14,8 @@ from (
   from
     ContactDetails cd
   left join GmcDetails gmc on (gmc.id = cd.id)
+  and gmc.id is not null
+  and not lower(gmc.id) = "unknown"
```

Comment:
`gmc.id` is the primary key and it equals the person id, so we have to use the column `gmcNumber` here.